README.md

```diff
@@ -1,43 +1,58 @@
 
-```sh
-npx create-turbo@latest -e with-yarn
-```
+To use ZooKyc, Gerry first goes to ZooGov (in `pod-issuer-client/app/gov`, `pod-issuer-server`), and ZooGov issues him an ID POD signed by ZooGov. Gerry then goes to ZooDeel (in `pod-issuer-client/app/deel`, `pod-issuer-server`), and ZooDeel issues him a Paystub POD signed by ZooDeel. All the PODs have a owner field with Gerry's Semaphore public identity.
+
+Gerry then goes to ZooKyc with his ID POD, Paystub POD, and Semaphore identity. ZooKyc creates a ZK proof that Gerry has a valid ID, and has monthly income over $X amount, without leaking other information. ZooKyc is a browser only app. In fact, Gerry can take the code from ZooKyc and run it himself.
+Now with the proof generated by ZooKyc, Gerry goes back to ZooLender (in `gpc-verifier-client`). ZooLender verifies the proof, and lends Gerry the money.
+
+## Example code
+
+- How to issue PODs: [code](https://github.com/proofcarryingdata/zukyc/blob/main/apps/pod-issuer-server/src/routes/debug.ts#L92-L114)
+- Proof request (what you want to prove, including proof configuration, and optionally membership lists, external nullifier and watermark): [code](https://github.com/proofcarryingdata/zukyc/blob/main/apps/gpc-verifier/app/hooks/useProofRequest.ts)
+- How to create GPC proofs (based on the proof request): [code](https://github.com/proofcarryingdata/zukyc/blob/main/apps/gpc-prover-client/app/util/generateProof.ts)
+- How to verify GPC proofs: [code](https://github.com/proofcarryingdata/zukyc/blob/main/apps/gpc-verifier/app/util/verifyProof.ts)
 
 ## What's inside?
 
-This Turborepo includes the following packages/apps:
+This Turborepo includes the following apps:
 
-### Apps and Packages
+### Apps
 
-- `docs`: a [Next.js](https://nextjs.org/) app
-- `web`: another [Next.js](https://nextjs.org/) app
-- `@repo/ui`: a stub React component library shared by both `web` and `docs` applications
-- `@repo/eslint-config`: `eslint` configurations (includes `eslint-config-next` and `eslint-config-prettier`)
-- `@repo/typescript-config`: `tsconfig.json`s used throughout the monorepo
+- `pod-issuer-client`: http://localhost:3000, deployed at https://zukyc-issuer-client.vercel.app. It includes:
+  - [ZooGov](https://zukyc-issuer-client.vercel.app/gov): Govenment ID POD issuer.
+  - [ZooDeel](https://zukyc-issuer-client.vercel.app/deel): Paystub POD issuer.
+  - [ZooAdmin](https://zukyc-issuer-client.vercel.app/debug): For debugging purposes, it allows you to generate ID PODs and Paystub PODs with the information you provide.
+- `pod-issuer-server`: http://localhost:8080, deployed at https://zukyc-issuer-server.vercel.app.
+  - This is the Express server for pod-issuer-client. It provides the APIs to issues PODs to the user.
```

Comment:
Maybe it's worth explaining why this is in a server?  I.e. in a real use case the server has the user database and holds the signing key.